data/political/2022-2032-political-districts/index.html

```diff
@@ -76,6 +76,8 @@ <h5>Related Resources</h5>
       <ul class="dotless">
```

Comment:
**thought**: should we rename this to political districts so we don't have to modify it every time?

Reply:
I like what you're saying.  it's gonna blow up some stuff with existing links on this site and also the sgid index.  Are you up for a little adventure (with the sgid index spreadsheet again and a fetch for my PR)?

Reply:
Yah, certainly